### PR TITLE
feat(profiling): add link to release

### PR DESCRIPTION
**Pull request by JonasBa**
Add link to release from side drawer.

### CODE REVIEW

File: static/app/types/profiling.d.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 declare namespace Profiling {
+  type Release = import('sentry/types').Release;
```
Comment (eliashussary) [inline static/app/types/profiling.d.ts:2]: TIL you can assign a type w/ and import

Comment (JonasBa) [reply static/app/types/profiling.d.ts:2]: I tried to actually import type from outside the namespace declaration and all hell broke loose and it stopped working. After a quick search it turns out this is the recommended way

File: static/app/components/profiling/FrameStack/profileDetails.tsx [hunk 2]
```diff
@@ -44,11 +46,36 @@ function renderValue(
...
+  // if only one project is selected in global header and release has it (second condition will prevent false positives like -1)
+  if (
+    selection.projects.length === 1 &&
+    release.projects.map(p => p.id).includes(selection.projects[0])
+  ) {
+    return selection.projects[0];
+  }
+
+  // project selector on release detail page will pick it up
+  return undefined;
```
[the remark this answers is not in the document]
Comment (JonasBa) [reply static/app/components/profiling/FrameStack/profileDetails.tsx:70]: Ah, I just copy pasted this. Basically if there is no param the UI will open a project selector

**Review by eliashussary — APPROVED** [review]

File: static/app/components/profiling/FrameStack/profileDetails.tsx [hunk 3]
```diff
@@ -230,6 +282,7 @@ const PROFILE_DETAILS_KEY: Record<string, string> = {
   [t('version')]: 'version',
   [t('duration')]: 'durationNS',
   [t('threads')]: 'threads',
+  [t('release')]: 'release',
```
Comment (Zylphrex) [inline static/app/components/profiling/FrameStack/profileDetails.tsx]: You can take the `version` off now, we won't send it to the frontend anymore

**Pull request merged**